_posts/0000-01-02-galiepst.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
---
layout: slide
title: "Welcome to our second slide!"
---
something new
Use the left arrow to go back!